src/java.desktop/share/classes/javax/swing/AbstractButton.java

```diff
@@ -2146,9 +2146,8 @@ protected void init(String text, Icon icon) {
 
 
     /**
-     * This is overridden to return false if the current <code>Icon</code>'s
-     * <code>Image</code> is not equal to the
-     * passed in <code>Image</code> <code>img</code>.
+     * Returns true if the button icon for present button state is an instance of ImageIcon
+     * and the image it contains is same as the <code>image</code>.
```

Comment:
boolean value should be put under <code>true</code>...also you should add "<code>false</code> otherwise"

Reply:
change added